lightning/src/ln/channelmanager.rs

```diff
@@ -4827,6 +4880,53 @@ where
 			)
 	}
 
+	#[cfg(async_payments)]
+	fn check_refresh_async_receive_offer(&self) {
```

Comment:
My view on channelmanager is that it manages the channels. Relatively low level operations. I think that async offers live on a higher level, a level above basic channel management. Shouldn't this be reflected in code? 

This is a drive-by high level comment.

Reply:
Good point, I believe this will move into `OffersMessageFlow` once #3639 lands!